build.gradle

```diff
@@ -1,10 +1,11 @@
 plugins {
+	id 'java'
 	id 'org.springframework.boot' version '2.4.3'
 	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'java'
-	id 'com.palantir.docker' version '0.26.0'
-	id "org.sonarqube" version "2.7"
+	id 'jacoco'
+	id 'org.sonarqube' version '2.7'
```

Comment:
What about a fresh version (https://mvnrepository.com/artifact/org.sonarsource.scanner.gradle/sonarqube-gradle-plugin), is it compatible with our Sonar server?

Reply:
Update done